src/main/java/uk/gov/hmcts/divorce/common/model/CaseData.java

```diff
@@ -88,31 +88,16 @@
     @CCD(access = {DefaultAccess.class})
     private Applicant applicant2 = new Applicant();
 
-    @CCD(
-        label = "Help with fees reference",
-        regex = "([Hh][Ww][Ff]-?)?[0-9a-zA-Z]{3}-?[0-9a-zA-Z]{3}$",
-        access = {DefaultAccess.class}
-    )
-    private String helpWithFeesReferenceNumber;
-
-    @CCD(
-        label = "Need help with fees?",
-        access = {DefaultAccess.class}
-    )
-    private YesOrNo helpWithFeesNeedHelp;
+    @JsonUnwrapped(prefix = "helpWithFees")
+    @CCD(access = {DefaultAccess.class})
+    private HelpWithFees helpWithFees;
```

Comment:
should this have ` @Builder.Default` and should we initialise this ?

Reply:
I thought about it but it didn't seem necessary. There weren't any bits of code that assumed this would be set